content/docs/clouds/aws/get-started/next-steps.md

```diff
@@ -17,6 +17,20 @@ aliases:
 
 Congrats! You've deployed your first project on AWS with Pulumi. Here are some next steps, depending on your learning style.
 
+## Try Pulumi ESC (Environments, Secrets, and Configuration)
+
+In this guide, you used [stack configuration](/docs/concepts/config/) and environment variables to configure a single project and stack. Did you know you can also configure multiple Pulumi projects, and even other applications, with [Pulumi ESC](/docs/esc/)?
+
+Pulumi ESC introduces the concept of _environments_ --- managed collections of static and dynamic settings that you can use to configure any project, stack, application, or service, including with short-lived cloud credentials through OpenID Connect.
```

Comment:
I feel like you should start this paragraph off by describing Pulumi ESC. Something like, "Pulumi ESC is a cetralized secrets management and orchestration service. It introduces the concept..."

Reply:
Adjusted intro